modules/core/simulation/proposals.go

```diff
@@ -0,0 +1,42 @@
+package simulation
+
+import (
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+)
+
+// Simulation operation weights constants
+const (
+	DefaultWeightMsgUpdateParams int = 100
+
+	OpWeightMsgUpdateParams = "op_weight_msg_update_params" // #nosec
+)
+
+// ProposalMsgs defines the module weighted proposals' contents
+func ProposalMsgs() []simtypes.WeightedProposalMsg {
+	return []simtypes.WeightedProposalMsg{
+		simulation.NewWeightedProposalMsg(
+			OpWeightMsgUpdateParams,
+			DefaultWeightMsgUpdateParams,
+			SimulateMsgUpdateParams,
+		),
```

Comment:
Same comment about whether we should add a second element to the slice using [`MsgUpdateParams` for 03-connection](https://github.com/cosmos/ibc-go/blob/5ca37ef6e56a98683cf2b3b1570619dc9b322977/proto/ibc/core/connection/v1/tx.proto#L131)?